doc-content/enterprise-only/release-notes/rn-7.10.1-known-issues-ref.adoc

```diff
@@ -0,0 +1,76 @@
+[id='rn-7.10.1-known-issues-ref']
+= Known issues in {PRODUCT} 7.10.1
+
+This section lists known issues with {PRODUCT} 7.10.1.
+
+ifdef::PAM[]
+
+== Process Designer
+
+.Custom task thumbnails are not consistent [https://issues.redhat.com/browse/RHPAM-3496[RHPAM-3496]]
+
+Issue: Custom task thumbnails are not consistent in both project and case project.
+
+Steps to reproduce:
+
+. Create a case project.
+. In case project, create both process and case.
+. Save the changes and close the process.
+. Navigate to the project settings and add all the custom tasks.
+. Save the changes.
+. Check the process.
++
+If the issue is not visible try to close and reopen the asset.
+. Close the process.
+. Remove all the custom tasks and save the changes.
```

Comment:
s/Remove all the custom tasks and save the changes./Remove all of the custom tasks and save the changes.

Reply:
Done